haystack/document_store/elasticsearch.py

```diff
@@ -987,14 +1001,28 @@ def delete_documents(self, index: Optional[str] = None, filters: Optional[Dict[s
         if self.refresh_type == "wait_for":
             time.sleep(2)
 
-class OpenDistroElasticsearchDocumentStore(ElasticsearchDocumentStore):
+
+class OpenSearchDocumentStore(ElasticsearchDocumentStore):
     """
-    Document Store using the Open Distro for Elasticsearch. It is compatible with the AWS Elasticsearch Service.
+    Document Store using OpenSearch (https://opensearch.org/). It is compatible with the AWS Elasticsearch Service.
 
     In addition to native Elasticsearch query & filtering, it provides efficient vector similarity search using
     the KNN plugin that can scale to a large number of documents.
     """
 
+    def __init__(self, **kwargs):
```

Comment:
It might be more readable to have explicit arguments(with their corresponding defaults).